src/FormStateApp.tsx

```diff
@@ -1,6 +1,7 @@
 import { Observer } from "mobx-react";
 import { AuthorInput } from "src/formStateDomain";
-import { FieldState, ObjectConfig, required, useFormState } from "src/index";
+import {FieldState, ObjectConfig, ObjectState, required, useFormState} from "src/index";
```

Comment:
Nit: looks like prettier didn't run

Reply:
Adding the husky + lint-staged packages to help this in the future.

Reply:
Great!